packages/asset-swapper/src/utils/market_operation_utils/orders.ts

```diff
@@ -412,7 +412,7 @@ function createNativeOrder(fill: CollapsedFill): OptimizedMarketOrder {
 }
 
 export function createSignedOrdersFromRfqtIndicativeQuotes(
-    quotes: RfqtIndicativeQuoteResponse[],
+    quotes: IndicativeQuote[],
```

Comment:
Why `IndicativeQuote` instead of `RFQTIndicativeQuote`?

Reply:
Good catch, thank you, fixed in https://github.com/0xProject/0x-monorepo/pull/2582/commits/c18f3f0b339adf3c2f315f363a8a6cf366f71104